torch/_inductor/codegen/cpp_utils.py

```diff
@@ -222,7 +222,7 @@ def depends_on(self, itervar: sympy.Symbol):
 
 class CppPrinter(ExprPrinter):
     def _print_Integer(self, expr):
-        return f"{int(expr)}LL" if _IS_WINDOWS else f"{int(expr)}L"
+        return f"{int(expr)}L"
```

Comment:
Don't we want to keep this (and maybe add mac)?  Otherwise large constants will be truncated on windows.

Reply:
I added LL on MacOS, but it failed.

Reply:
Let try it again: https://github.com/pytorch/pytorch/pull/133873